jcl/src/openj9.jvm/share/classes/com/ibm/jvm/InvalidDumpOptionException.java

```diff
@@ -31,8 +33,19 @@
 
 	private static final long serialVersionUID = -507148799087920306L;
 
+	/**
+	 * @param message description of error
+	 */
 	public InvalidDumpOptionException(String message) {
 		super(message);
 	}
+
+	/**
+	 * @param cause root exception
+	 */
+	public InvalidDumpOptionException(InvalidDumpOptionExceptionBase cause) {
+		super(cause.getMessage());
+		initCause(cause);
```

Comment:
I don't think we want the base exception chained here.

Reply:
We would lose the stack trace, unless I copy it over.

Reply:
I don't think there will be much information in `cause`. If you insist, please use the super constructor with signature `(String, Throwable)`.

Reply:
Please change this to `super(cause.getMessage(), cause);`.

Reply:
Done.